Resources/Prototypes/_NF/Recipes/Cooking/meal_recipes.yml

```
@@ -292,6 +321,19 @@
   recipeType:
   - Assembler
 
+- type: microwaveMealRecipe
+  id: RecipeBatteredCuredCarp
+  name: battered fish recipe
+  result: FoodCuredCarpBattered
+  time: 2 # ~5 seconds for three
+  reagents:
+    Flour: 1.5 # ~1/3 of a click
+    Egg: 2 # 1/3 of an egg
```

```suggestion
  time: 5
  reagents:
    Flour: 5
    Egg: 6
```
Equivalent with a steak/chicken, not a cutlet.